### PR TITLE
Added copy&paste-able build requirements

### PR DESCRIPTION
**Pull request by Zeerix**

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ On a Debian system those are the following, excluding packages with priority ess
 - dosfstools
 - binutils
 
+Install them as root:
+```
+aptitude install git curl bzip2 zip xz-utils gnupg kpartx dosfstools binutils
```
Comment (goranche) [inline BUILD.md]: are we sure it will always be run on a debian compatible machine?

**Comment by Zeerix** [comment]
Of course not, but this is just a convenience. The package names that are listed above can also be different on other distributions.

**Comment by diederikdehaas** [comment]
`On Debian (based) systems you can install them as root or with sudo as follows:`
Seems more correct

**Comment by diederikdehaas** [comment]
`bc` has been added recently to v1.0.x but we'll fix that when we merge v1.0.x into v1.1.x

**Pull request merged**